modules/local/templates/anndatar_convert.R

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env Rscript
+
+# to use nf variables: "${meta.id}"
+
+# load libraries
+library(anndataR)
+
+# read input
+adata <- read_h5ad("${h5ad}")
+
+# convert to Rds
+obj <- adata\$to_Seurat()
```

Comment:
This should also write a SingleCellExperiment rds file. While Seurat and Bioconductor both use `rds` files, they use completely different data structures. AnndataR can generate both.